config/initializers/defra_ruby_exporters.rb

```diff
@@ -0,0 +1,18 @@
+module DefraRuby
```

Comment:
Just spotted this doesn't have a `# frozen_string_literal: true` comment at the top.

- think it should have one
- would this remove the need to use `.freeze`?
- more a question, wonder why rubicon isn't picking this up 🤔

Reply:
The freeze statements are redundant. This is fixed in 9105b48.